src/components/Header/Header.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import StyledHeader from './styles';
+
+const Header = (props) => {
+  const {
+    title,
+    subtitle,
+    thumbnail,
+    icon,
+    alignText,
+    divider,
+    dividerColor,
+    ...attributes
+  } = props;
+  const dividerClass = divider ? 'withDivider' : '';
+
+  return (
+    <StyledHeader {...attributes}>
+
+      <StyledHeader.ColumnLeft {...attributes}>
+        {thumbnail}
+      </StyledHeader.ColumnLeft>
+
+      <StyledHeader.ColumnRight align={alignText} {...attributes}>
```

Comment:
Could you normalize the name of the `align` prop? I think it got a little confusing because the component is accepting a prop called `alignText` and passing this prop as `align` for its children. My suggestion is to change this to keep only one name. Either align or alignText.

Reply:
@csilva2810 done! 😄